compiler/tflchef/proto/tflchef.proto

```diff
@@ -366,6 +366,12 @@ message SquaredDifferenceOptions {
   // None
 }
 
+message SVDFOptions {
+  optional int32 rank = 1 [default = 0];
+  optional Activation activation = 2 [default = NONE];
+  optional bool asymmetric_quantize_inputs = 3 [default = false];
+}
+
```

Comment:
For reviewers:
need to add this `SVDFOptions` since the `test.recipe` files use it.
Maybe it is necessary to add this in a separate PR?

Reply:
> Maybe it is necessary to add this in a separate PR?

Yes, that would be nice and rebase this PR after that lands :)

Reply:
Or if you like, you can land all tflchef codes with the proto.
My memories are not sure which one we should land first; tflchef or recipes...

Reply:
Yes, I think tflchef should land first, so this pr will be landed after landing this one #8284